### PR TITLE
Show modal with origin selector before creating localization

### PR DESCRIPTION
**Pull request by arthurperton**
As discussed, this PR shows a confirmation modal before creating a localization for an entry. The origin for the new localization will default to the first site, however in the modal you can choose a different origin if you like. 

Screenshot 2022-10-24 at 15 14 02

Feel free to update UI and text of course.

### CODE REVIEW

**Comment by arthurperton** [comment]
I suspected you wouldn’t like that too much 😊 They’re not needed for the fix and they make the diff larger right. I’ll revert them 👍🏻

**Comment by jasonvarga** [comment]
Thanks! Much easier to review the relevant parts.

**Review by jasonvarga — CHANGES_REQUESTED** [review]
If you only have 2 sites configured, you shouldn't get the modal at all since there's only one option to choose from.

**Comment by arthurperton** [comment]
Ah yes, totally.

File: resources/js/components/entries/PublishForm.vue [hunk 1]
```diff
@@ -599,8 +599,10 @@ export default {
 
             if (localization.exists) {
                 this.editLocalization(localization);
-            } else {
+            } else if (this.localizations.length > 2) {
```
[the remark this answers is not in the document]
Comment (jasonvarga) [reply resources/js/components/entries/PublishForm.vue:602]: The ability to cancel is a good enough reason to keep it. 👍

**Comment by tao** [comment]
Is there a way to disable the modal from showing up? I already have an event to only translate the root entry from ages ago. https://github.com/statamic/cms/issues/4287

**Comment by tao** [comment]
There needs to be a way to disable this before I can update, I can't have editors creating localisations from other languages besides English as the root.

**Comment by jasonvarga** [comment]
The modal will only show options that they would have been able to use before.

**Comment by tao** [comment]
There should be a way to support many different types of workflows with Multisite, the original issue was created back in Sep 2021 and we found a work-around, which I've been very happy with since then... now the pop-up is going to break the flow and confuse everyone if I update Statamic. 

Is there some Javascript we can add or custom Javascript I can run to disable this modal or auto-select English and suppress the pop-up?

**Comment by jasonvarga** [comment]
I see what you mean. You were forcing the origin on the server side. So in your case, they could pick another site in the modal, even though it wouldn't use it because you override it.

**Pull request merged**